Library/Formula/mailhog.rb

```diff
@@ -0,0 +1,171 @@
+      :revision => "36077cc82f0e00c479943a06173a600dedefd7a0"
+  end
+
+  go_resource "github.com/mailhog/http" do
+    url "https://github.com/mailhog/http.git",
+      :revision => "066a2dfd9d8bf7a94a2df4e1b4b9db3556b56bbb"
+  end
+
+  go_resource "github.com/mailhog/mhsendmail" do
+    url "https://github.com/mailhog/mhsendmail.git",
+      :revision => "0bba7051c64d8af0b436d0c47774d12a8684a3e1"
+  end
+
+  go_resource "github.com/mailhog/smtp" do
+    url "https://github.com/mailhog/smtp.git",
+      :revision => "8c087763ca1b9208b0890dab7aacd487c07ba06a"
+  end
+
+  go_resource "github.com/mailhog/storage" do
+    url "https://github.com/mailhog/storage.git",
+      :revision => "33a9b882525652a3e4b74b03716e0225de010ba4"
+  end
+
+  go_resource "github.com/mailhog/MailHog-Server" do
+    url "https://github.com/mailhog/MailHog-Server.git",
+      :revision => "1dfb973b724a309c18474fe032b1458c0d8654fc"
+  end
+
+  go_resource "github.com/mailhog/MailHog-UI" do
+    url "https://github.com/mailhog/MailHog-UI.git",
+      :revision => "ebdfa0f91fac40169ca376953d7c13f9d83cd01a"
+  end
+
+  go_resource "github.com/mailhog/MailHog" do
+    url "https://github.com/mailhog/MailHog.git",
+      :revision => "91a9d8afe119dc96a6c9acfa94c997765c824908"
+  end
+
+  go_resource "github.com/ogier/pflag" do
+    url "https://github.com/ogier/pflag.git",
+      :revision => "6f7159c3154e7cd4ab30f6cc9c58fa3fd0f22325"
+  end
+
+  go_resource "github.com/t-k/fluent-logger-golang" do
+    url "https://github.com/t-k/fluent-logger-golang.git",
+      :revision => "0f8ec08f2057a61574b6943e75045fffbeae894e"
+  end
+
+  go_resource "github.com/tinylib/msgp" do
+    url "https://github.com/tinylib/msgp.git",
+      :revision => "c46fbee0675f25770e5f340dc9bc22bf4a108ad4"
+  end
+
+  go_resource "golang.org/x/crypto" do
+    url "https://github.com/golang/crypto.git",
+      :revision => "c8b9e6388ef638d5a8a9d865c634befdc46a6784"
+  end
+
+  go_resource "github.com/philhofer/fwd" do
+    url "https://github.com/philhofer/fwd.git",
+      :revision => "8fd9a4b75098b2125fe442e48a3ffbf738254e13"
+  end
+
+  # Use file resource instead of revision until https://github.com/Homebrew/homebrew/issues/44828
+  # is resolved.
+  go_resource "labix.org/v2/mgo" do
+    url "https://github.com/go-mgo/mgo/archive/2e26580ebcca4eca6533535f2dd062cd6bff44ac.tar.gz"
+    sha256 "9ef389ccc5410e9cf54073b8db048739577c5ed8f2491cfd35f7e3a695082503"
+  end
+
+  def install
+    mkdir_p "#{buildpath}/src/github.com/mailhog/"
+    ln_s buildpath, "#{buildpath}/src/github.com/mailhog/MailHog"
+
+    ENV["GOPATH"] = buildpath
+    Language::Go.stage_deps resources, buildpath/"src"
+    system "go", "build", "-o", "MailHog"
+    bin.install "MailHog"
+  end
+
+  def caveats; <<-EOS.undent
+    To use MailHog as a sendmail replacement for PHP, add the
+    following line to php.ini file:
+
+      sendmail_path = #{HOMEBREW_PREFIX}/bin/MailHog sendmail test@test
+    EOS
+  end
```

Comment:
It seems that there's no `caveats` equivalent for info messages, will use regular `puts` in `def install` to inform users about sendmail replacement option.